src/components/Image.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import Img from 'next/image';
+
+const Image = (props) => {
+
+    const [error, setError] = useState(false);
+    
+    const {
+        src,
+        width = 240,
+        height = 240,
+        alt = '',
+        ...otherProps
+    } = props;
+    
+    // URL to use if the actual image fails to load.
+    const fallBackUrl = `https://via.placeholder.com/${width}x${height}?text=${alt}`;
```

Comment:
Lets update to using a static image, rather than being dependent on third part url.

Reply:
Which image can we use for fallback?

Reply:
May be we can get something from the design team. For now we can just keep an image with gray background( feel free to use what you like )